### ISSUE TITLE
Better Instructions Needed

### ISSUE THREAD

**Issue by bomalley** [awslabs/aws-sdk-react-native#18]
There are conflicting instruction on the various Amazon properties for how to get this working in React Native

[Announcing the AWS SDK for React Native](https://aws.amazon.com/blogs/mobile/announcing-the-aws-sdk-for-react-native/) has one set of instructions
[aws-sdk-react-native](https://github.com/awslabs/aws-sdk-react-native) at GitHub has different instructions

Both are very sparse and not detailed enough for me.

In particular, it's not very clear how the packages on this site are supposed to interact with the SDK files. What gets compiled? What gets linked to the Xcode project, etc.

Could someone please writeup a step-by-step set of instructions to get setup and Authenticate with Cognito on iOS?

Thanks!

**Comment by tioback** [comment]
Related to #15

**Comment by bomalley** [comment]
Agreed, though I believe this step is a precursor to #15.  Just the basics of how to install and compile properly are needed.  Then we can get in to how to actually use the SDK productively.

**Comment by minbi** [comment]
Thank you for providing valuable insight into your experience and use of the SDK. We will incorporate this into our improvement items.

**Comment by iamhainguyen** [comment]
@minbi what is the update on it? The instructions on both AWS blog and this repo are still unclear

**Comment by damathryx** [comment]
any update?
